room/compiler-processing-testing/src/test/java/androidx/room/compiler/processing/util/GeneratedCodeMatchTest.kt

```diff
@@ -111,4 +114,70 @@ class GeneratedCodeMatchTest internal constructor(
         )
         assertThat(result.exceptionOrNull()).hasMessageThat().contains(mismatch.toString())
     }
+
+    @Test
+    fun successfulGeneratedKotlinCodeMatch() {
+        // java environment will not generate kotlin files
+        if (runTest.toString() == "java") return
```

Comment:
```suggestion
        if (runTest.toString() == "java") throw AssumptionViolatedException()
```
so that test is reported as skippped instead of succeeded.